src/js/libgeo/GeoOps.js

```diff
@@ -2519,6 +2520,12 @@ geoOps._helper.initializeLine = function(el) {
 };
 
 
+geoOps.Poly = {};
+geoOps.Poly.kind = "Poly";
+geoOps.Poly.signature = "Ps";
+geoOps.Poly.updatePosition = noop;
```

Comment:
The `updatePosition` code should collect all the `homog` vectors from the arguments into a list. That's because one day we'll want to transform polygons (e.g. apply projective transformations to them), and then we can't derive those coordinates from the arguments without case distinctions.

Do we need points at infinity or via points or some such, in order to identify the line segments correctly after a projective transformation? If unsure we can postpose adding them, but I fear we are going to need them.